examples/todo/package.json

```diff
@@ -1,39 +1,40 @@
 {
   "private": true,
   "scripts": {
-    "start": "npm run create-token && babel-node server.js",
     "check": "flow check",
     "create-token": "node tools/create-token.js",
+    "start": "npm run create-token && babel-node server.js",
     "update-schema": "babel-node tools/updateSchema.js"
   },
   "dependencies": {
-    "@4c/babel-preset-4catalyzer": "^1.2.1",
-    "@4c/graphql-subscription-server": "link:../..",
-    "@babel/core": "^7.0.0-beta.41",
-    "@babel/node": "^7.0.0-beta.41",
-    "@babel/polyfill": "^7.0.0-beta.41",
-    "@babel/preset-flow": "^7.0.0-beta.41",
-    "babel-loader": "^7.1.4",
-    "babel-plugin-relay": "^1.4.1",
+    "@4c/babel-preset-4catalyzer": "^1.4.0",
+    "@4c/graphql-subscription-server": "../..",
```

Comment:
ugh i have no idea how to get this to work. i kept getting "duplicate graphql" errors with the `link`. with this, it works as long as i've nuked `node_modules` in the parent... i don't know how to make this work correctly with peer deps

Reply:
I do the install then manually delete graphql from the example node_modules, it's not a great workflow

Reply:
you can’t link graphql too?

Reply:
the plan was really to not link, and jsut use the last published version for folks taht want to run it, but practically it's better as an integration test for us so whatever works...maybe we should add a command to https://github.com/4Catalyzer/file-butler for 🔥 extra graphql copies down